data-raw/tranform_inputs_complete.R

```diff
@@ -1199,3 +1219,4 @@ swp_exports <- list(biop_2008_2009 = swp_exports_2008_2009,
                     run_of_river = swp_exports_run_of_river)
 
 usethis::use_data(swp_exports, overwrite = TRUE)
+
```

Comment:
Can you please update where you use = instead of assigned (<- ) looks like it happens on line 146.